tests/PHPStan/Levels/data/acceptTypes.php

```diff
@@ -180,6 +180,12 @@ public function benevolentUnionNotReported(array $strings)
 		}
 	}
 
+	/**
+	 * @param string|int|array $union
+	 */
+	public function partlySupportedUnion($union) {
+		$imploded = implode('abc', $union);
```

Comment:
1) Add a new test at the end so that the line numbers do not shift like that. It's hard to review that
2) Add another example that reports an error on level 5.